Zend/tests/bug73960.phpt

```diff
@@ -0,0 +1,15 @@
+--TEST--
+Bug #73960: Leak with instance method calling static method with referenced return
+--FILE--
```

Comment:
It's better to add  opcache.enable=0 here, I can not reproduce this when it is enabled